views/boilerplate/template/error.mustache

```diff
@@ -1,7 +1,7 @@
 {{>boilerplate/template/partial/header}}
 
-<h1>{{errorTitle}}</h1>
```

Comment:
I would recommend we take advantage of Charcoal's enhanced error handlers to provide some useful information in debug mode.

Check out Terrebonne's error template controller and view as an example.

```
{{# debug }}
<pre>
{{& htmlErrorDetails }}
</pre>
{{/ debug }}
{{^ debug }}
<h1>{{# _t }}Page Not Found{{/ _t }}</h1>
<p>{{# _t }}The page you are looking for could not be found.{{/ _t }}</p>
{{/ debug }}
```